db-engine/src/main/java/com/splicemachine/db/iapi/error/StandardException.java

```diff
@@ -712,14 +712,13 @@ public String getErrorProperty(String type) {
 	*/
 	public String toString() {
 		String msg = getMessage();
-		StringBuffer buf = new StringBuffer(256);
-		buf.append("ERROR ").append(getSQLState()).append(": ").append(msg).append("\n");
+		String buf = "ERROR " + getSQLState() + ": " + msg + "\n" +
+				"Splice Machine Release: " + System.getProperty(Property.SPLICE_RELEASE) + "\n" +
+				"Splice Machine Version Hash: " + System.getProperty(Property.SPLICE_VERSION_HASH) + "\n" +
+				"Splice Machine Build Time: " + System.getProperty(Property.SPLICE_BUILD_TIME);
 		// Include the Splice Machine version/release information.
```

Comment:
String concat is converted to a StringBuilder op (non-synchronized) and supposedly easier to read.